src/app/(proper_react)/(redesign)/(public)/how-it-works/HowItWorksView.module.scss

```diff
@@ -0,0 +1,280 @@
+@import "../../../../tokens";
+
+.brokersEmphasis {
+  display: inline;
+  color: $color-violet-50;
+}
+.breachesEmphasis {
+  display: inline;
+  color: $color-purple-70;
+}
+
+.header {
+  padding: $layout-lg $spacing-md;
+  display: flex;
+  justify-content: center;
+  span {
+    width: 100%;
+    max-width: $content-xl;
+  }
+  @media screen and (min-width: $content-md) {
+    padding: $layout-lg;
+  }
+  .title {
+    font: $text-title-lg;
+    font-family: "Inter", sans-serif;
+    color: $color-grey-60;
+  }
+  p {
+    color: $color-grey-40;
+    font: $text-body-xl;
+  }
+}
+.sectionWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  &.grayBG {
+    background: $color-grey-05;
+  }
+  ul {
+    list-style-type: none;
+    padding: 0;
+  }
+  .intro {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    max-width: $content-xl;
+    padding: $spacing-sm;
+    @media screen and (min-width: $content-md) {
+      padding: $spacing-2xl 0;
+      gap: $spacing-lg;
+    }
+    .introContent {
+      gap: $spacing-md;
+      display: flex;
+      flex-direction: column;
+      h2 {
+        color: $color-grey-60;
+        font: $text-title-md;
+        font-family: "Inter", sans-serif;
+      }
+      p {
+        color: $color-grey-40;
+        font: $text-body-xl;
+      }
+      .horizontalRule {
+        border: 1px solid $color-grey-20;
+        margin: $spacing-md 0;
+      }
+    }
+    .introCTA {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      justify-content: center;
+      gap: $spacing-lg;
+      padding-inline: $spacing-md;
+      padding-block-start: $spacing-xl;
+      width: 100%;
+      h3 {
+        color: $color-grey-50;
+        font: $text-title-xs;
+        font-family: "Inter", sans-serif;
+        flex: auto;
+        max-width: $content-xl;
+        @media screen and (min-width: $screen-md) {
+          width: $content-sm;
+        }
+      }
+    }
+  }
+  .step {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    max-width: $content-xl;
+    &.reversedStep {
+      flex-direction: column-reverse;
+    }
+    .stepTextContainer {
+      max-width: $content-sm;
+      padding: $spacing-sm;
+      .stepTitle {
+        font: $text-title-2xs;
+        font-family: "Inter", sans-serif;
+        padding-block-start: $spacing-xl;
+        text-transform: uppercase;
```

Comment:
This is [not reliable across locales](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues). I get that it's only English for now, but it will bite us back when we don't expect it.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4656/commits/58ee62a9a95cd9abe1cc00dd84b218795061b6d6